qiskit/opflow/primitive_ops/pauli_sum_op.py

```diff
@@ -291,22 +300,30 @@ def to_instruction(self) -> Instruction:
     def to_pauli_op(self, massive: bool = False) -> OperatorBase:
         from .pauli_op import PauliOp
 
-        def to_real(x):
-            return x.real if np.isreal(x) else x
+        def chop_real_imag(x):
```

Comment:
How about using `np.real_if_close`, e.g., `np.real_if_close(x, tol=1e-12)`?
https://numpy.org/doc/stable/reference/generated/numpy.real_if_close.html#numpy.real_if_close

Reply:
Thanks.
This code is previous code...
https://github.com/Qiskit/qiskit-terra/blob/03d5965e823a8943da138358b4b5c21b9c474944/qiskit/opflow/legacy/weighted_pauli_operator.py#L443-L450

I will use it.